### PR TITLE
Update EIP-1: Add first two external origins

### PR DESCRIPTION
**Pull request by SamWilsn**
If we're okay with this format, I'll update `eipw` and get the csl-json to render into a human readable citation.

### CODE REVIEW

File: EIPS/eip-1.md [hunk 1]
```diff
@@ -188,7 +188,97 @@ EIPs may have a `requires` header, indicating the EIP numbers that this EIP depe
 
 ## Linking to External Resources
 
-Links to external resources **SHOULD NOT** be included. External resources may disappear, move, or change unexpectedly.
+Other than the specific exceptions listed below, links to external resources **SHOULD NOT** be included. External resources may disappear, move, or change unexpectedly.
+
+The process governing permitted external resources is described in [EIP-5757](./eip-5757.md).
+
+### Consensus Layer Specifications
```
Comment (lightclient) [inline EIPS/eip-1.md:195]: Might be good to explicitly state that CL spec links need to be anchored to a specific commit.

**Review by lightclient — COMMENTED** [review]
One nit; generally LGTM though.

File: EIPS/eip-1.md [hunk 2]
```diff
@@ -188,7 +188,97 @@ EIPs may have a `requires` header, indicating the EIP numbers that this EIP depe
...
+
+```markdown
+[Beacon Chain](https://github.com/ethereum/consensus-specs/blob/26695a9fdb747ecbe4f0bb9812fedbc402e5e18c/specs/sharding/beacon-chain.md)
+```
+
+Which renders to:
+
+[Beacon Chain](https://github.com/ethereum/consensus-specs/blob/26695a9fdb747ecbe4f0bb9812fedbc402e5e18c/specs/sharding/beacon-chain.md)
+
+Permitted Consensus Layer Specifications URLs must match this regular expression:
```
Comment (SamWilsn) [inline EIPS/eip-1.md:207]:
```suggestion
Permitted Consensus Layer Specifications URLs must anchor to a specific commit, and so must match this regular expression:
```

**Review by lightclient — APPROVED** [review]

File: EIPS/eip-1.md [hunk 3]
```diff
@@ -188,7 +188,97 @@ EIPs may have a `requires` header, indicating the EIP numbers that this EIP depe
...
+        ]
+      },
+      "URL": "https://sly-hub.invalid/00.0000/a00000-000-0000-y",
+      "custom": {
+        "additional-urls": [
+          "https://example.com/an-interesting-article.pdf"
+        ]
+      }
+    }
+    ```
```
Comment (Pandapip1) [inline EIPS/eip-1.md:277]: Nit: don't indent the code block.
```suggestion
```csl-json
{
  "type": "article",
  "id": 1,
  "author": [
    {
      "family": "Jameson",
      "given": "Hudson"
    }
  ],
  "DOI": "00.0000/a00000-000-0000-y",
  "title": "An Interesting Article",
  "original-date": {
    "date-parts": [
      [2022, 12, 31]
    ]
  },
  "URL": "https://sly-hub.invalid/00.0000/a00000-000-0000-y",
  "custom": {
    "additional-urls": [
      "https://example.com/an-interesting-article.pdf"
    ]
  }
}
```
```

Comment (SamWilsn) [reply EIPS/eip-1.md:277]: For the code block to be part of the footnote it needs to be indented. It's very non-standard markdown :(

Comment (Pandapip1) [reply EIPS/eip-1.md:277]: Does Commonmark support footnotes? The EIPs website doesn't use GFM.

Comment (SamWilsn) [reply EIPS/eip-1.md:277]: If we use `kramdown`, I think they'll work: https://kramdown.gettalong.org/quickref.html#footnotes

Comment (Pandapip1) [reply EIPS/eip-1.md:277]: We switched to commonmark because it runs 25% quicker, and is run with the longest-running check.

**Comment by kdenhartog** [comment]
Looks like this one is moving pretty fast for the first draft. Anyone mind if I add sections for W3C/IETF as well to this or do people think it's easier to do this in a separate EIP? I'm cautious that we're going to end up dispersing governance across many EIPs and somethings may be lost, so think doing these all in a single EIP would be better. Wdyt?

**Comment by xinbenlv** [comment]
Thank you for awesome effort!

With such complex details about origin, shall it be better to be a seperate json or yaml structura file to be easily integrat with EIPW and website rendering? 

@SamWilsn @Pandapip1

**Comment by Pandapip1** [comment]
> Do people think it's easier to do this in a separate EIP?

Probably - that's what EIP-5757 suggests.

File: EIPS/eip-1.md [hunk 4]
```diff
@@ -188,7 +188,97 @@ EIPs may have a `requires` header, indicating the EIP numbers that this EIP depe
 
 ## Linking to External Resources
```
Comment (xinbenlv) [inline EIPS/eip-1.md:189]: Propose to add [ethereum/devp2p](https://github.com/ethereum/devp2p) as allowable origin

**Comment by lightclient** [comment]
What else are we waiting for here?

**Review by Pandapip1 — CHANGES_REQUESTED** [review]
I still would like confirmation that footnotes work.

**Comment by SamWilsn** [comment]
> I still would like confirmation that footnotes work.

How's this: https://samwilsn.github.io/EIPs/EIPS/eip-1

**Review by Pandapip1 — APPROVED** [review]
Footnotes are a bit awkward, but LGTM

**Comment by lightclient** [comment]
Pressing the button.

**Pull request merged**